docs/index.rst

```diff
@@ -237,7 +242,9 @@ The `issues <https://github.com/solegalli/feature_engine/issues/>`_ and
    outliers/index
    creation/index
    selection/index
+   preprocessing/index
    wrappers/index
+   preprocessing/index
 
```

Comment:
this entry is duplicated and it is causing the error in the tests:

/home/circleci/project/docs/index.rst:234:duplicated entry found in toctree: preprocessing/index